Source/Core/Common/Timer.cpp

```diff
@@ -261,4 +261,13 @@ double Timer::GetDoubleTime()
 	return TmpTime;
 }
 
+std::string Timer::GetDateTimeFormatted(double time)
+{
+	time_t seconds = (time_t)time + (38 * 365 * 24 * 60 * 60);
```

Comment:
It's from [here](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Common/Timer.cpp#L248), which is directly written to the savestate header. Changing it would make old savestates display incorrect dates, which I guess isn't too big of a deal in the long run but I don't think it's hurting anything to just leave it.

But yeah I can extract that into a constant.